iotilecore/RELEASE.md

```diff
@@ -2,6 +2,10 @@
 
 All major changes in each released version of `iotile-core` are listed here.
 
+## 5.0.6
```

Comment:
Let's call this HEAD and not create a new version

Reply:
Got it